src/jbtoc.ts

```diff
@@ -178,7 +173,7 @@ async function glob_files(pattern: string): Promise<any> {
     }
 
     const files = await response.json();
-    let result = [];
+    const result = [];
     for (const file of files) {
       if (file.type === 'file') {
         result.push(file.path);
```

Comment:
We're pushing to a `const` array here -- this might be my lack of experience with TS, but shouldn't this just break?